app/src/main/java/com/monkeyteam/chimpagne/viewmodels/FindEventsViewModel.kt

```diff
@@ -69,6 +69,30 @@ class FindEventsViewModel(database: Database) : ViewModel() {
     }
   }
 
+  fun fetchAroundLocation(onSuccess: () -> Unit = {}, onFailure: (Exception) -> Unit = {}) {
+    eventManager.getAllEventsByFilterAroundLocation(
+        _uiState.value.selectedLocation!!,
+        9999999999.0,

```

Comment:
Magic value. Is it really useful to display such far events ? The fetched events are supposed to be *close* to the user.